internals/cli/cmd_exec.go

```diff
@@ -71,7 +71,7 @@ stderr are output locally.
 To avoid confusion, exec options may be separated from the command and its
 arguments using "--", for example:
 
-pebble exec --timeout 10s -- echo -n foo bar
+	exec --timeout 10s -- echo -n foo bar
```

Comment:
I think removing the program here make these examples worse; it's a bit confusing without the program name prefix, because you'd never actually run it that way. Any way to include it, or do we have an ordering problem where `Personality` has to be set (by the main program) before init time?

If there's an ordering problem, can we do the default `addCommand` calls at parse / run time rather than init time?

Reply:
This might seem like a trivial issue to fix, and I've proposed many options in the past. However, a brief discussion sufficed to conclude that the simplest approach was to try and let go of self-references in help strings.

It's not ideal, but it's the solution with less impact in overall complexity of the code base.

Reply:
Fair enough. I guess we could also do something like `<program> exec --timeout ...`, but I'm not sure that's any clearer. Oh well, as is is okay.

Reply:
I would still prefer a placeholder that can be used in general as the options without the program itself doesn't feel very clean.